### DISCUSSION TITLE
Osu! Lazer update on android

### DISCUSSION THREAD

**Discussion by Fikrienz** [ppy/osu#26882] [Q&A] [answered]
Can someone tell me how to update osu! Lazer on android? Do i need to uninstall my osu! Lazer and download from osu.ppy.sh web?
![Screenshot_2024-01-31-18-21-07-91_439a3fec0400f8974d35eed09a31f914](https://github.com/ppy/osu/assets/157803695/d4fca641-2406-4058-a1b9-336643adb69d)

**Comment by ZenitH-AT** [comment, marked as answer]
The Android APK for this version isn't ready yet. You'll need to check back later.

Also you don't need to uninstall anything. Just open the new APK when it's available.
